rust/private/rust.bzl

```
@@ -378,18 +379,17 @@ def _rust_test_impl(ctx):
     deps = transform_deps(ctx.attr.deps)
     proc_macro_deps = transform_deps(ctx.attr.proc_macro_deps + get_import_macro_deps(ctx))
 
+    prefix_test_output_files = ctx.attr._prefix_test_output_files[BuildSettingInfo].value
+
     if ctx.attr.crate:
         # Target is building the crate in `test` config
         crate = ctx.attr.crate[rust_common.crate_info] if rust_common.crate_info in ctx.attr.crate else ctx.attr.crate[rust_common.test_crate_info].crate
 
-        output_hash = determine_output_hash(crate.root, ctx.label)
-        output = ctx.actions.declare_file(
-            "test-%s/%s%s" % (
-                output_hash,
-                ctx.label.name,
-                toolchain.binary_ext,
-            ),
-        )
+        output_name = "%s%s" % (ctx.label.name, toolchain.binary_ext)
```

I personally prefer format strings unless there's a performance reason to do otherwise. Is there?
```suggestion
        output_name = "{}{}".format(ctx.label.name, toolchain.binary_ext)
```

---

Done.